README.md

```diff
@@ -1,21 +1,35 @@
 # Pyroscope nodejs package
 
 ## Running Pyroscope server
+
 In order to send data from your node application to Pyroscope, you'll first need to run the Pyroscope server. If on a mac you can simply do this with
+
 ```
 # install pyroscope
 brew install pyroscope-io/brew/pyroscope
 
 # start pyroscope server:
 pyroscope server
 ```
+
 or if not then see [the documentation](https://github.com/pyroscope-io/pyroscope#add-pyroscope-server-locally-in-2-steps) for more info on how to start the server.
 
+## Environment Variables Configuration
+
+The agent can be configured using following environment variables:
+
+- **PYROSCOPE_SAMPLING_INTERVAL** - The interval in milliseconds between samples.
+- **PYROSCOPE_SAMPLING_DURATION** - The duration in milliseconds for which you want to collect a sample.
+- **PYROSCOPE_SERVER_ADDRESS** - The address of the Pyroscope server. The default is http://localhost:4040
+- **PYROSCOPE_APPLICATION_NAME** - The application name used when uploading profiling data.
+- **PYROSCOPE_AUTH_TOKEN** - The authorization token used to upload profiling data.
```

Comment:
A table like https://github.com/pyroscope-io/pyroscope-lambda-extension#configuration is easier to read IMO, not sure about description, but having a default value explicit is very useful IMO.